packages/k4lcioreader/package.py

```diff
@@ -11,6 +11,7 @@ class K4lcioreader(CMakePackage, Key4hepPackage):
     maintainers = ['mirguest']
 
     version('master', branch='master')
+    version('0.3.2', sha256='dac76172d484faa23ca170fc89b4cfec6ce56777cd0c20b2b608838824582447')
```

Comment:
This does not correspond to the tag on github, which is `00.03.02` and due to that will fail while fetching. The previous tags are not padded with zeros. Could you edit the tag on github to follow the convention of the previous tags?

Reply:
Hi @vvolkl and @fdplacido , I rename the tag name to "v0.3.2".